src/extension.ts

```diff
@@ -127,10 +127,18 @@ export function activate(context: ExtensionContext) {
 
 		const errorhandler = new LsErrorHandler(serverOptions)
 
+		let compilerVenv = workspace.getConfiguration('inmanta').compilerVenv;
+		if (!compilerVenv) {
+			compilerVenv = Uri.joinPath(context.globalStorageUri, "compiler-venv").fsPath;
```

Comment:
Is it ok that this directory is shared among projects? I think it is but I'm not sure. @arnaudsjs what do you think? (I had missed during the core PR review that it was an absolute path.)

Reply:
I don't think it's a good idea to share this venv among projects. Imagine you open two vscode windows using a different inmanta project. If both projects have conflicting requirements defined in the requirements.txt files of their dependencies, both projects will start fighting. Also, I am not convinced that the venv library support concurrent access. It wouldn't surprise me if concurrent access on the same venv would corrupt the venv. 

@sanderr I don't get the remark about the absolute path. The PR on core converts a relative path to an absolute path, as such that a change in the cwd wouldn't cause side effects. What is the link with this comment?

Reply:
Now that I think about it the remark about the absolute path does not have anything to do with this. I noticed it while typing the comment and it confused my thinking I guess. And indeed, it's not an absolute path, it's relative to the current working directory, while I had assumed relative to the project root.
Anyway, you can safely ignore that part. I agree with @arnaudsjs' assessment.